README.adoc

```diff
@@ -228,6 +172,69 @@ The reply should be as follows:
 ----
 ====
 
+
+
+== Preparing to Build the Application
+
+To package and run the application, we need to provide an external MySQL database rather than using Spring Boot Docker Compose Support.
+For this task, we can reuse the `compose.yaml` provided with a few modifications.
+First, modify the `ports` entry in `compose.yaml` to be `3306:3306`.
+Second, add a `container_name` of `guide-mysql`.
+After these steps, the `compose.yaml` file should be:
```

Comment:
Add a blank line before this line, to separate the (implicitly defined) list from the content around it.